### PR TITLE
Development_Setup.rst: Update Documentation generation instructions

### PR DESCRIPTION
**Pull request by Uditgulati**
Updates the documentation generation command from ``make html`` to ``python3 setup.py docs``.

Closes coala/coala#5062

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Uditgulati** [comment]
@newbazz  done

File: docs/Developers/Development_Setup.rst [hunk 1]
```diff
@@ -127,5 +127,6 @@ You should run this command before trying to build the documentation:
     (coala-venv)$ pip3 install -r docs-requirements.txt
 
 Once you have done so, you can build the documentation by entering the docs
-directory and running ``make``. The documentation on the coala website is in
+directory and running ``python setup.py docs``. The documentation
```
Comment (Makman2) [inline docs/Developers/Development_Setup.rst]: please use `python3` :+1:

**Comment by Makman2** [comment]
unack 6ace2766c2694e5b9d98fd97b83d7eaaaded2f48

File: docs/Developers/Development_Setup.rst (hunk 1 above)
Comment (Uditgulati) [reply docs/Developers/Development_Setup.rst]: fixed :)

**Comment by Makman2** [comment]
ack f8e7bb73234d379b0f5359cc28f8b1e324fa7210

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Comment by EverWinter23** [comment]
YES!!!!!

**Pull request merged**